### PR TITLE
feat: add citibikebymk app

### PR DESCRIPTION
**Pull request by kleban31**
This app displays available bikes as well as docks for the chosen Citibike station, which the user can search by name. This differs from the existing Citibike app in that it shows available docks, which can help with trip planning.

### CODE REVIEW

**Comment by kleban31** [comment]
I have read the CLA Document and I hereby sign the CLA

File: apps/citibikebymk/citibike_by_mk.star [hunk 1]
```diff
@@ -0,0 +1,97 @@
+load("encoding/base64.star", "base64")
+load("encoding/json.star", "json")
+load("http.star", "http")
+load("render.star", "render")
+load("schema.star", "schema")
+
+url_stat = "https://gbfs.citibikenyc.com/gbfs/en/station_information.json"
+url_bikes = "https://gbfs.citibikenyc.com/gbfs/en/station_status.json"
+resp = http.get(url_bikes)
```
Comment (matslina) [inline apps/citibikebymk/citibike_by_mk.star:9]: We'll need some caching around these calls to keep request rates to the API at a reasonable level.

File: apps/citibikebymk/citibike_by_mk.star [hunk 2]
```diff
@@ -0,0 +1,97 @@
+load("encoding/base64.star", "base64")
+load("encoding/json.star", "json")
+load("http.star", "http")
+load("render.star", "render")
+load("schema.star", "schema")
+
```
Comment (matslina) [inline apps/citibikebymk/citibike_by_mk.star:6]: Code executing at this level (as opposed to in or below `main()`) will only be run once when the app is compiled. In this case you'll end up with stale data pretty quickly.

**Review by matslina — CHANGES_REQUESTED** [review]
Thanks for the PR! A few issues to address before we can merge.

File: apps/citibikebymk/citibike_by_mk.star (hunk 1 above)
Comment (kleban31) [reply apps/citibikebymk/citibike_by_mk.star:9]: Thanks @matslina. Where can I find documentation on how to set up caching for API calls in my starlark project?

File: apps/citibikebymk/citibike_by_mk.star (hunk 2 above)
Comment (kleban31) [reply apps/citibikebymk/citibike_by_mk.star:6]: Got it. I'll fix.

File: apps/citibikebymk/citibike_by_mk.star [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
...FRXr0+JM2p25a7072f5pnz9bg/Ia1+bCxV+11+TVj6v8A+G4rb/oT5f8AwYD/AON0f8NxW3/Qny/+DAf/ABuvlCiun/WvOP8An9/5LH/5E5P9ReHv+gf/AMnn/wDJH1f/AMNw2v8A0KE3/gwH/wAbrnbz9tjxG1xKbTQNLig3kxrMZHYLngEhhk4xzgfSvnKis58T5vUVnX+5RX5I1p8E5BSd44ZP1lJ/nJntGqftcfEG+vJJba7s9NhYjbBBaI6rgAHBcMeTzye9chr/AMcPHXiSS5N54mvxHcJ5ckFvKYYipXaRsTAwR147n1rhqK8qtmuPr3VWvJr/ABO33bHuYfI8rwrTo4aEWuvKr/fa5JNdTXLZmleU9cuxP86jooryz29tgooooGFFFFABRRRQAUUUUAFFFFABRRRQAUUUUAFFFFABRRRQAUUUUAFFFFABRRRQAUUUUAFFFFABRRRQAUUUUAFFFFABRRRQAUUUUAFFFFABRRRQAUUUUAFFFFABRRRQAUUUUAFFFFABRRRQAUUUUAFFFFABRRRQAUUUUAFFFFABSqxVgykhgcgjqKSigDd0Px54j8MrMNJ13UNOE2DILa5dN+M4zg84yfzrttM/ac+I2l2kVumvmdI84a6t45XbJJ5ZlJPX16V5ZRXdRx2Lw6tRqyj6No8zEZXgMY74ihCb84p/mj6CtP20vF8FvDHNpWkXLoqq8rJIrSEDliA+AT14GOelbej/ALbt9H539q+GLe4zjyvsdy0WOud24Nnt0x3r5ior16fEmbU3dV387P8ANHgVeDshqpqWFS9G1+TR9X/8NxW3/Qny/wDgwH/xuj/huK2/6E+X/wAGA/8AjdfKFFdH+tecf8/v/JY//InJ/qLw9/0D/wDk8/8A5I+r/wDhuG1/6FCb/wAGA/8AjdczJ+2x4pK/JoWkKf8AaEp/9nFfO1FZVOJ83qb138lFfkjanwVkFL4cMn6uT/Ns9o1X9rj4g315JLbXdnpsLY2wQWiOq4ABwXDHk88nvXIa98cPHXiSS5N54mvxHcJ5ckFvL5MRUrtI2JgYI68c5PrXDUV5dbNcfXuqteTX+J2+7Y9vD5HleFadHDQTXXlV/vtckmuprlszSySnrl2J/nUdFFeWe3tsFFFFAwooooAKKKKACiiigAooooAKKKKACiiigAooooAKKKKAP//Z"""
```
[the remark this answers is not in the document]
Comment (kleban31) [reply apps/citibikebymk/citibike_by_mk.star:19]: Got it. I'll look into sizing the image down and removing the exif data

**Comment by kleban31** [comment]
@matslina Made suggested changes. I'm not sure if I used the caching module correctly. There's not api key (public data).

**Comment by kleban31** [comment]
Done setting up caching. Ready for review.

File: apps/citibikebymk/citibike_by_mk.star (hunk 1 above)
Comment (kleban31) [reply apps/citibikebymk/citibike_by_mk.star]: Nevermind. Figured it out.

**Review by matslina — APPROVED** [review]

**Pull request merged**